src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -45,6 +46,7 @@ protected function configure()
                 new InputArgument('name', InputArgument::OPTIONAL, 'The bundle name or the extension alias'),
                 new InputArgument('path', InputArgument::OPTIONAL, 'The configuration option path'),
                 new InputOption('resolve-env', null, InputOption::VALUE_NONE, 'Display resolved environment variable values instead of placeholders'),
+                new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (yaml or json)', 'yaml'),
```

Comment:
We could switch the default value by checking if yaml is available, that would improve the experience for projects without Yaml.

`default: class_exists(Yaml::class) ? 'yaml' : 'json'`.